updater/schema.py

```diff
@@ -20,6 +20,9 @@ def checkSchemaVersion(dataDirectory):
 				if row[0] == "schema-version":
 					schemaVersionLocal = int(row[1])
 					break
+	except IOError:
+		print("error: the data repository has no file named meta.tsv", file = sys.stderr)
+		sys.exit(1)
```

Comment:
👋 @IngoS11 ! Thanks for fixing this issue!

I wonder, couldn't we just assume the default `schemaVersion` in case of an `IOError`? That means we would set:
```
schemaVersionLocal = schemaVersion
```
here.

What do you think?